httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/HttpRFC7578Multipart.java

```diff
@@ -138,7 +137,7 @@ public byte[] decode(final byte[] bytes) throws DecoderException {
                 final int b = bytes[i];
                 if (b == ESCAPE_CHAR) {
                     if (i >= bytes.length - 2) {
-                        throw new DecoderException("Invalid URL encoding: too short");
+                        throw new IllegalArgumentException("Invalid URL encoding: too short");
```

Comment:
What? URL encoding? This must be a typo, no?

Reply:
That's the original message, but agreed it seems wrong. Fixed.